modules/ppcp-settings/src/Data/Definition/PaymentMethodsDependenciesDefinition.php

```diff
@@ -107,6 +107,27 @@ public function get_setting_dependencies(): array {
 		);
 	}
 
+	/**
+	 * Get payment method value dependencies for a specific method
+	 *
+	 * @return array Value dependencies for the method or empty array if none exist
```

Comment:
Sorry but what does the "Value dependencies for the method" mean? maybe "A list of method dependencies" ?
failing to understand what value in"method value dependencies" mean

Reply:
This is to differentiate from `get_payment_method_dependencies()`.

Essentially, `get_payment_method_dependencies()` allows you to define enabled gateways as a requirement for enabling a gateway.

`get_payment_method_value_dependencies()` allows you to define an enabled/disabled requirement for enabling a gateway.

`get_payment_method_dependencies()` can potentially be refactored in the future to be handled by `get_payment_method_value_dependencies()`, having these dependencies in a single place.